### ISSUE TITLE
Won’t open

### ISSUE THREAD

**Issue by chemicalsam** [gpodder/gpodder#1038]
I downloaded gpodder (windows 10). Clicking on the exe does nothing.

**Comment by elelay** [comment]
Which version (installer or portable)? Some antivirus flagged gPodder as malware. Maybe it's what's blocks it?

**Comment by chemicalsam** [comment]
> Which version (installer or portable)? Some antivirus flagged gPodder as malware. Maybe it's what's blocks it?

Installer
Using malwarebytes and defender. I didn’t get any sort of pop up when clicking on them. Just nothing happens at all.

**Comment by alienzoo** [comment]
Same issue - clicking the exe does nothing. Regular installer, windows 11, only Defender (scanning the exe with Defender produces no warning). MD5 and SHA256 match those in another thread. Running 3.11.4 installer exe immediately pops up request for admin password.

212a88b68469e9b83e920d950b260ce6a9eea3fae913e6a3bff9833ab31bf6a1 *windows-gpodder-3.11.5-installer.exe
2e09083e15a5f189464b708fad52009d *windows-gpodder-3.11.5-installer.exe

**Comment by masoudd** [comment]
> Same issue 3.11.5 - clicking the exe does nothing. Regular installer, windows 11, only Defender (scanning the exe with Defender produces no warning). MD5 and SHA256 match those in another thread. Running 3.11.4 installer exe immediately pops up request for admin password. Tried the download twice to be sure.
> 
> 212a88b68469e9b83e920d950b260ce6a9eea3fae913e6a3bff9833ab31bf6a1 *windows-gpodder-3.11.5-installer.exe 2e09083e15a5f189464b708fad52009d *windows-gpodder-3.11.5-installer.exe

Can you run the exe in cmd to see if it prints any errors? Open start menu, type `cmd` and hit enter, drag the `windows-gpodder-3.11.5-installer.exe` file into the cmd window and hit enter.

**Comment by alienzoo** [comment]
The exe prints "Access denied" whereas the 3.11.4 installed run from the
command line pops up the expected UAC window prompting for an admin
password and only prints "Access denied" when I click "No".

--Alex

**Comment by masoudd** [comment]
> The exe prints "Access denied" whereas the 3.11.4 installer run from the command line pops up the expected UAC window prompting for an admin password and only prints "Access denied" when I click "No".

Hmm I don't have access to windows 11 to test. Can you run it by rightclick -> Run as adminstrator ?

**Comment by alienzoo** [comment]
What a bizarre and inconsistent world of drivel Windows developers have created. I ran a series of tests:

1. Right-click run as administrator: no output, nothing appears to happen.
2. From cmd window runs as administrator, run the installer: pops up window to uninstall 3.11.4 (hooray).
3. Logged in as admin account, right-click installer: nothing happens.
4. From admin cmd window, run installer: pops up notification that SmartScreen wants to block the app (in a window that's too small).
5. As an admin accout, right-click and run as administrator: same SmartScreen notification.
6. Re-ran Defender scan, still showed nothing so no clue what "Smart"Screen is on about
7. Despite never overriding the SmartScreen check, suddenly all methods of running the installer now work as expected. UAC window pops up running the installer as a normal user, and the uninstall 3.11.4 window pops up when run as an admin user.

Which of these steps made stuff work normally, I have no clue. I now have 3.11.5 installed successfully, thanks for the help. I would suggest if anyone else has trouble that option 2 above was the quickest route to something that worked.

**Comment by masoudd** [comment]
I think you can submit the file to microsoft to add to [their whitelist here](https://learn.microsoft.com/en-us/windows/security/operating-system-security/virus-and-threat-protection/microsoft-defender-smartscreen/#submit-files-to-microsoft-defender-smartscreen-for-review). I submitted the `windows-gpodder-3.11.5-installer.exe` and [result](https://www.microsoft.com/en-us/wdsi/submission/c0c0a310-c8b3-4c3a-9ba5-2ddaf1b30b07) seems to be "No malware detected". 🤷‍♂️

**Comment by elelay** [comment]
Thank your both for the investigation and solution.
@masoudd is submission for whitelisting possible via a non-windows os? I might add it to the release procedure.

**Comment by auouymous** [comment]
@elelay Should we add a note next to the download link on gpodder.org? I assume most Windows users would be notified by the gpodder client and sent to gpodder.org to download.

**Comment by elelay** [comment]
Yes, good idea. It would be to let people know that in case of error running the installer on windows 11 they might try to start cmd or powershell as administrator and drag-and-drop the installer to the terminal, then press enter? Is that the correct procedure?
I hope the false positive will be removed from windows defender in a short while.

**Comment by masoudd** [comment]
>is submission for whitelisting possible via a non-windows os
Yes. You just need to make a microsoft account and submit files from a web browser.

I'm not sure if we can preemptively submit files to microsoft. The submit form assumes the files you want to submit are either malicious but not detected or detected as malicious incorrectly.

>start cmd or powershell as administrator and drag-and-drop the installer to the terminal, then press enter? Is that the correct procedure?

Yes I think that's what did the trick but we need more testing. I think adding a note is a good idea either way

**Comment by alienzoo** [comment]
For clarity, the process that first worked **_for me_** was to be a regular user, open a standard "Command Prompt" as Administrator (right click option), change to the directory where the installer resided, and type its name using command expansion. That means that the command ran in the same directory as the installer resided. I have no clue if that was important. Drag-and-drop might well have worked (had no idea it was a thing), but that process would run the installer while located in the root directory. Whether that would have mattered, I do not know and since my Windows has now changed how it responds to the installer, I have no way of testing.

Given that Windows responded in at least three different ways to actions that should have led to identical or near identical outcomes, I simply do not know what made this specific process work differently from the others. Nor how it would behave for other people. Wish I could be more helpful, but:
    1. Defender **_never_** recognised the installer as any kind of threat (scanned the installer at least twice, once as regular user and once as admin).
    2. "Smart"Screen eventually, and only in certain specific circustances I listed above, decided to show a completely uninformative warning completely contradicting Defender.

If SmartScreen thought there was a problem then I simply do not understand how any procedure could have worked, nor why all of the procedures above didn't show the SmartScreen warning. Nor why everything suddenly just worked, since I'm convinced I never whitelisted the installer.

The **_only_** way I ever saw the SmartScreen warning was to be logged in as an Admin user. Using the same process as above ("Command Prompt" right-click, run as Admin, cd to installer directory and run it) generated the SmartScreen warning.

I can find no trace in the information that "Windows Security" provides that the installer was ever blocked, either as my regular user or as an Admin user. Nor that I whitelisted it in any way.
